Microsoft.Toolkit.Mvvm/ComponentModel/ObservableRecipient.cs

```diff
@@ -7,15 +7,9 @@
 // This file is inspired from the MvvmLight library (lbugnion/MvvmLight),
 // more info in ThirdPartyNotices.txt in the root of the project.
 
-// ================================= NOTE =================================
```

Comment:
Reason for the note to be removed, I thought this was important?

Reply:
Oh, that was because the comment was just a leftover for an earlier implementation of the source generator project when I was also keeping that file as a separate copy, but now I'm just linking that so it's already in sync, as it's just the same file. The only file that's actually a separate copy is `INotifyPropertyChanged`, which still has the comment on it 🙂